snippets/3.4/rpm.html.in

```diff
@@ -2,9 +2,12 @@
   <p>
     We provide a single package containing the server and the client tools. For a
     standard server setup, download the package <strong>Server</strong> and
-    install it using the package manager. The <strong>Client Tools</strong>
-    package only contain the tools, like arangosh, arangodump, but nor the
-    server.
+    install it using the package manager. You do not need the client tools.
+
+    The <strong>Client Tools</strong> package only contain the tools, like arangosh,
+    arangodump, but nor the server. Use this in case you have already an arangod
```

Comment:
Use this in case you're already running ArangoDB on a different server ...

Reply:
done